### DISCUSSION TITLE
Adding CacheDirectory for faster access to the browser.

### DISCUSSION THREAD

**Discussion by mckabue** [hardkoded/puppeteer-sharp#1845] [Ideas]
Scenario: Azure function runs for a limited amount of time (10 mins  - Consumption plan), and one has to download the browser each time the app starts!

> Azure Functions/Apps with *Headless Chromium*
>  -  [Headless Chromium ONLY works in the Linux Consumption plan](https://anthonychu.ca/post/azure-functions-headless-chromium-puppeteer-playwright/)
>  -  [Linux consumption apps ONLY Run-From-Package.](https://stackoverflow.com/a/56035713/3563013)
>  -  [You can ONLY use "shared mounted storage" or `Path.GetTempPath()`.](https://anthonychu.ca/post/azure-functions-puppeteer-pdf-razor-template/)

What do you think about including this?

See commit at: https://github.com/McKabue/puppeteer-sharp/commit/48be8d119eb5c5c31ec21140e8e5aa236347dbd6

**Comment by kblok** [comment]
I like that!

**Reply by mckabue** [reply]
https://github.com/hardkoded/puppeteer-sharp/pull/1849
